1.9/usage/pods/quick-start.md

```diff
@@ -0,0 +1,31 @@
+---
+post_title: Quick Start
+menu_order: 0
+---
+
+# Prerequisites: A DC/OS Cluster, DC/OS CLI installed
+
+# Launch a Pod from the DC/OS CLI
+
+1. Create a JSON application definition with the following code. In this example, we will call the file `simple-pod.json`.
+
```

Comment:
Indent this code block the same amount as the numbered step. Otherwise the step numbers are restarted.